staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1627,6 +1627,8 @@ message ContainerStats {
     MemoryUsage memory = 3;
     // Usage of the writable layer.
     FilesystemUsage writable_layer = 4;
+    // Usage of the Block IO
+    BlockIOUsage block_io = 5;
```

Comment:
Is this possibly relevant to the windows stats object as well?

cc @marosset

Reply:
were we going to deprecate the duplicate object definitions?

Reply:
no the path forward is to have them defined for each OS to allow customized stats for each (when eventually appropriate)

Reply:
thx :-) let it be written!